pkg/cmd/root.go

```diff
@@ -32,6 +32,7 @@ var gardenConfig string
 var pathGardenHome string
 var sessionID string
 var debugSwitch bool
+var getTargetInfo map[string]string = make(map[string]string)
```

Comment:
- Type can be omitted
- Rename to `targetInfo` as this is not a getter

```suggestion
var targetInfo = make(map[string]string)
```

Reply:
fixed